src/storage/benches/bench_bloom_filter.rs

```diff
@@ -0,0 +1,80 @@
+use criterion::{criterion_group, criterion_main, Criterion};
+use rand::rngs::SmallRng;
+use rand::{RngCore, SeedableRng};
+use risingwave_storage::hummock::sstable::{Bloom, InMemWriter, Sstable, SstableBuilder};
+use rocksdb_util::{FilterBitsBuilderWrapper, FilterBitsReaderWrapper, FilterType};
+
+const TEST_COUNT: usize = 1024 * 16;
+
+fn bench_bloom_filter_read(c: &mut Criterion) {
+    let mut origin_data = Vec::with_capacity(TEST_COUNT);
+    let mut data = Vec::with_capacity(TEST_COUNT);
+    for idx in 0..TEST_COUNT {
+        let key = format!("test_000001_{:08}", idx);
+        data.push(Sstable::hash_for_bloom_filter(key.as_bytes()));
+        origin_data.push(key);
+    }
+    let mut rng = SmallRng::seed_from_u64(10244021u64);
+    let data = Bloom::build_from_key_hashes(&data[..(TEST_COUNT / 2)], 14);
```

Comment:
What does 14 here means?👀

Reply:
It is the default bits_per_key. I must make sure that the size of  the bits_per_key is same for both algorithm

Reply:
I'm not familiar with ribbon filter. But for bloom filter, bits_per_key is caculuated based on the given `number of keys` and `expected FPR`. `number of keys` is decided by the SST KV counts. `expected FPR` is a configurable parameter. It seems here the experiment is based on fixed `number of keys` and `bits_per_key`. Should we use `number of keys` and `expected FPR` instead?